src/Driver/Pdo/AbstractPdoCommand.php

```diff
@@ -52,8 +57,12 @@ abstract class AbstractPdoCommand extends AbstractCommand implements PdoCommandI
      */
     protected PDOStatement|null $pdoStatement = null;
 
-    public function __construct(protected PdoConnectionInterface $db)
+    /**
+     * @param PdoConnectionInterface $db The PDO database connection to use.
+     */
+    public function __construct(PdoConnectionInterface $db)
     {
+        parent::__construct($db);
```

Comment:
```suggestion
    public function __construct(protected readonly PdoConnectionInterface $db)
    {
```

Ok, if we override the parent property we should not initialize the parent property.